tox.ini

```diff
@@ -1,50 +1,14 @@
 [tox]
-envlist = py{37,38,39}
+envlist = py{38,39,310,311,312}
 
 [testenv]
-whitelist_externals = echo make
 deps =
     -rrequirements.txt
-    flake8
 allowlist_externals =
```

Comment:
We can drop the allowlist_externals. We only use pytest in commands which is installed by tox (so not external)